librz/debug/debug.c

```diff
@@ -1756,3 +1756,16 @@ RZ_API void rz_debug_bp_rebase(RzDebug *dbg, ut64 old_base, ut64 new_base) {
 		bp->delta = bp->addr - dbg->bp->baddr;
 	}
 }
+
+/**
+ * @brief Switches to the first thread in the given debug context.
+ * @param debug The debug context.
+ */
+RZ_API void rz_debug_switch_to_first_thread(RZ_NONNULL RzDebug *debug) {
```

Comment:
```
RZ_API void rz_debug_switch_to_first_thread(RZ_NONNULL RzDebug *debug) {
```
you define `debug` as nonnull, but you never do the API check using `rz_return_if_fail`

Reply:
sorry, I missed it.